api/v1alpha3/common_types.go

```diff
@@ -105,6 +105,8 @@ type ObjectMeta struct {
 	// Cannot be updated.
 	// More info: http://kubernetes.io/docs/user-guide/identifiers#names
 	// +optional
+	//
+	// Deprecated: This field has no function and is going to be removed in a next release.
```

Comment:
Question: should we backport the deprecation warning only to the release 0.3 branch?

Reply:
We should, yeah